src/Microsoft.TestPlatform.CrossPlatEngine/TestSession/ProxyTestSessionManager.cs

```diff
@@ -20,6 +22,13 @@ namespace Microsoft.VisualStudio.TestPlatform.CrossPlatEngine;
 /// </summary>
 public class ProxyTestSessionManager : IProxyTestSessionManager
 {
+    private enum TestSessionState : int
```

Comment:
It seems like we are using the enum name and not its value for the telemetry, is there a reason to manually assign values?

Reply:
Not really, removing the values. Since the enum is private we can always add them later should we need them.